Tests/XcodeGenKitTests/FixtureTests.swift

```diff
@@ -59,5 +59,15 @@ func fixtureTests() {
                 try expect(variableGroup.children.filter { $0 == refs.first?.reference }.count) == 1
             }
         }
+
+        $0.it("generates scheme execution actions") {
```

Comment:
As a note for the future, tests here aren't really required as if the generated project changes the test will fail. Someone just added some more explicit checks afterwards above. No harm in having this here though 👍